PackHistorian/View/PackNameConverter.cs

```diff
@@ -328,6 +328,26 @@ class PackNameConverter : IValueConverter {
           { Locale.thTH, "Scholomance Academy" },
         }
       },
+      {
+        470, new Dictionary<Locale, string>() {
+          { Locale.enUS, "Golden Pack" },
+          { Locale.enGB, "Golden Pack" },
+          { Locale.frFR, "Golden Pack" },
+          { Locale.deDE, "Golden Pack" },
+          { Locale.koKR, "Golden Pack" },
+          { Locale.esES, "Golden Pack" },
+          { Locale.esMX, "Golden Pack" },
+          { Locale.ruRU, "Golden Pack" },
+          { Locale.zhTW, "Golden Pack" },
+          { Locale.zhCN, "Golden Pack" },
+          { Locale.itIT, "Golden Pack" },
+          { Locale.ptBR, "Golden Pack" },
+          { Locale.plPL, "Golden Pack" },
+          { Locale.ptPT, "Golden Pack" },
+          { Locale.jaJP, "Golden Pack" },
+          { Locale.thTH, "Golden Pack" },
+        }
+      },
```

Comment:
470 number is not exactly a "Golden pack". This is a special pack of Hunter cards.